scripts/index-services.py

```diff
@@ -64,7 +67,8 @@ def __call__(self, service):
 
     @backoff.on_exception(backoff.expo, apiclient.HTTPError, max_tries=5)
     def index_service(self, client, service):
-        if service['status'] == 'published':
+        if service['status'] == 'published' and \
```

Comment:
I don't think we need the additional check here. Once the `find_services` call is updated it should only return the services for the given frameworks.